.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'State: Unconfirmed, Type: Bug'
+assignees: m273d15 
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Log files**
+
+Please provide the corresponding log files, but **think of substituting sensitive data** you don't want to publish.
+
+You can find the log files in the following locations:
+* IntelliJ IDEA (with default config), where `.IdeaXXXXXXXX` is the directory containing the IDE files (e.g. `.IdeaIC2019.1`).</br>
```

Comment:
As this is mostly read in the editor and not displayed (as it should be removed by the user), I don't think the formatting (e.g. `<br>` or specified link references `[text](link)`) makes much sense here and just makes it harder to read.

Markings for fields (e.g. using `) is still sensible in my eyes.